modules/openapi-generator/src/main/resources/python-aiohttp/requirements.mustache

```diff
@@ -1,3 +1,4 @@
-connexion[aiohttp,swagger-ui] == 2.6.0
+# 2.3 is the last version that supports python 3.5
+connexion[aiohttp,swagger-ui] >= 2.3.0
```

Comment:
Hey so connexion only limits installation versions by supported versions of python in 3.6.0
per https://github.com/zalando/connexion/commit/f31ac615bfc8a3c696c5ebe53f3c57e5234ff6f8
Please:
- pin the connexion version for python 3.5
- pin the werkzeug version for python 3.5 (per your other work)
- pin `connexion >= 2.6.0; python_version>="3.6"` to cover higher versions of python

Reply:
Thank you for fixing this